terraform/rtmp-server/bootstrap.sh

```diff
@@ -162,7 +168,7 @@ cd ~
 until [[ `ifconfig tun1 >/dev/null 2>&1; echo $?` -eq 0 ]]; do
   sleep 1
 done
-echo -n `ifconfig tun1 | grep 'inet6 fd' | awk '{print $2'}` > ~/client-keys/rtmp_yggdrasil
+echo -n `ifconfig tun1 | grep -E 'inet6 2[0-9a-fA-F]{2}:' | awk '{print $2}'` > ~/client-keys/rtmp_yggdrasil
```

Comment:
Although i have never run into this issue, There is an assumption we are making that Yggdrasil will actually be on Tun1.

I found you can define the interface name it will be using
`sudo sed -i "s/IfName: auto/IfName: ygg0/" /etc/yggdrasil.conf`
That way instead of `tun1`  Yggdrasil will be on `ygg0`

PS: Great regex, going to port it over to prototype :+1:

Reply:
Also general comment:

should move to  `"$(xxx)"` instead of the \`xx\`
[SC2006](https://github.com/koalaman/shellcheck/wiki/SC2006): Use $(..) instead of legacy `..`.

Reply:
Thanks for the comment re: SC2006

Opened a new issue to address the more general problem of needing shellcheck.